src/Cli/ConfigGenerator.cs

```diff
@@ -691,64 +655,47 @@ public static bool TryUpdateExistingEntity(UpdateOptions options, ref string run
         /// <param name="fieldsToExclude">fields that are excluded from the operation permission.</param>
         /// <param name="existingOperations">operation items present in the config.</param>
         /// <returns>Array of updated operation objects</returns>
-        private static object[] GetUpdatedOperationArray(string[] newOperations,
-                                                        Policy? newPolicy,
-                                                        Field? newFields,
-                                                        IDictionary<Operation, PermissionOperation> existingOperations)
+        private static EntityAction[] GetUpdatedOperationArray(string[] newOperations,
+                                                        EntityActionPolicy newPolicy,
+                                                        EntityActionFields? newFields,
+                                                        IDictionary<EntityActionOperation, EntityAction> existingOperations)
         {
-            Dictionary<Operation, PermissionOperation> updatedOperations = new();
+            Dictionary<EntityActionOperation, EntityAction> updatedOperations = new();
 
-            Policy? existingPolicy = null;
-            Field? existingFields = null;
+            EntityActionPolicy existingPolicy = new(null, null);
```

Comment:
nit: looks like the EntityActionPolicy constructor will assign null here for the two arguments, so can commit the null values.